src/test/java/stepsDefinition/MainMemberOnlyAt80years.java

```diff
@@ -0,0 +1,104 @@
+package stepsDefinition;
+
+import PageObjects.*;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import net.thucydides.core.annotations.Steps;
+import java.util.Random;
+
+import java.util.Random;
+
+
+public class MainMemberOnlyAt80years {
+
+    @Steps
+    HomePage homePage;
+    @Steps
+    MemberPage memberPage;
+
+    @Steps
+    AdditionalMembers additionalMembers;
+    @Steps
+    BeneficiaryDetails beneficiaryDetails;
+    @Steps
+    PayerDetails payerDetails;
+
+    @Given(": Open browser and select funeral1")
+    public void open_browser_and_select_funeral1() {
+        homePage.open();
+        homePage.ClickBuyNowFuneralPlans();
+       // memberPage.EnterEmail("21mjhnbgh09@gmail.com");
+        Random randomGenerator = new Random();
+        int randomInt = randomGenerator.nextInt(1000);
+        memberPage.EnterEmail("Address"+ randomInt +"@gmail.com");
```

Comment:
Well done on implementing this, can you create it as method so it can be used by any class